lib/ansible/modules/database/postgresql/postgresql_idx.py

```diff
@@ -229,21 +229,17 @@
   sample: true
 '''
 
-import traceback
-
-PSYCOPG2_IMP_ERR = None
 try:
-    import psycopg2
-    HAS_PSYCOPG2 = True
-except ImportError:
-    HAS_PSYCOPG2 = False
-    PSYCOPG2_IMP_ERR = traceback.format_exc()
+    from psycopg2.extras import DictCursor
+except Exception:
```

Comment:
Any reason this line has changed?
```suggestion
except ImportError:
```

Reply:
@gundalow , thank you! I just fixed it everywhere